CMakeLists.txt

```diff
@@ -100,6 +100,13 @@ if(NOT CUKE_DISABLE_GTEST)
 endif()
 
 #
+# cgreen
+#
+if(NOT CUKE_DISABLE_CGREEN)
+    find_package(cgreen QUIET)
```

Comment:
`QUIET` suppresses both success and failure messages. For consistency with the other `find_package` calls, until , I suggest adding a more standard `Findcgreen.cmake`.

Feel free to pull [this commit](https://github.com/paoloambrosio/cucumber-cpp/commit/973e44406058a26702a9bdde8601761953995f05) on my personal repository that adds it on top of your PR.